torch/_inductor/ir.py

```diff
@@ -7235,24 +7242,25 @@ def create(
         assert len(cond_outputs[0].get_size()) == 0, cond_outputs
 
         assert (
-            len(operands) > 0
+            len(carried_inputs) > 0
```

Comment:
Actually, we can fetch the device from additional_inputs, too. So maybe just check that we have any input at least once?